src/Symfony/Component/Mailer/Transport.php

```diff
@@ -100,14 +100,19 @@ private static function createTransport(string $dsn, EventDispatcherInterface $d
                     throw new \LogicException('Unable to send emails via Mailgun as the bridge is not installed. Try running "composer require symfony/mailgun-mailer".');
                 }
 
+                $region = Mailgun\MailgunRegionConfiguration::REGION_DEFAULT;
+                if (!empty($query['region'])) {
+                    $region = $query['region'];
+                }
```

Comment:
can be replaced by `$region = $query['region'] ?? null;`

Reply:
I would prefer to keep explicitly declared default value as soon as it cause misunderstanding before why there is default region is US